x-pack/plugins/ml/public/application/components/anomaly_results_view_selector/anomaly_results_view_selector.test.tsx

```diff
@@ -16,8 +16,15 @@ import { AnomalyResultsViewSelector } from './index';
 
 jest.mock('../../contexts/kibana', () => {
   return {
-    useMlUrlGenerator: () => ({
-      createUrl: jest.fn(),
+    useMlLocator: () => ({
```

Comment:
Looks like useMlLocator returns `share.url.locators.get(ML_APP_LOCATOR)`. Should this be part of the mocks exported out by `src/plugins/share/public/mocks.ts`?

Reply:
Done.